process-engine/process_engine_meta
# ✨ Add processInstanceId to written metrics

## Changes

1. Add `processInstanceId` to written metrics
2. Only store token payloads instead of the whole ProcessToken object

## Issues

Part of https://github.com/process-engine/process_engine_runtime/issues/367

PR: #206

## How to test the changes

- Run the tests.
- Examine the generated metric files
- See that they no longer store a full ProcessTokenObject, but only a payload
- See that each entry will contain a processInstanceId
